tools/samba/illumina_suite/alpha_diversity.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="picrust2_plots" name="Statistical representation" version="@TOOL_VERSION_SWARM@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
+    <description>of functional predictions</description>
+    <xrefs>
+        <xref type="bio.tools"></xref>
+    </xrefs>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="figaro_edam"/>
+    <requirements>
+        <requirement type="package" version="1.1.4">r-dplyr</requirement>
+        <requirement type="package" version="1.1.4">r-tidyr</requirement>
+        <requirement type="package" version="1.5.1">r-stringr</requirement>
+        <requirement type="package" version="0.8.9">r-vegan</requirement>
+        <requirement type="package" version="0.8.9">r-phyloseq</requirement>
+        <requirement type="package" version="1.5.1">r-reshape2</requirement>
+        <requirement type="package" version="0.8.9">r-rstatix</requirement>
+        <requirement type="package" version="0.8.9">r-grid</requirement>
+        <requirement type="package" version="0.8.9">r-rcolorbrewer</requirement>
+        <requirement type="package" version="0.8.9">r-ggpubr</requirement>
+        <requirement type="package" version="0.8.9">r-ggplot2</requirement>
```

Comment:
TO DO put the real version of packages (for now it's just a bad copy paste) for those with real versions I put the latest one available on anaconda. 
Maybe better if I take the ones from the environment you created ?